src/main/java/com/hierynomus/msdtyp/FileTime.java

```diff
@@ -0,0 +1,59 @@
+package com.hierynomus.msdtyp;
```

Comment:
If you run `gradle licenseFormat` it will add the license header to this file, which should make the build green.